src/codehelp/formatter.vala

```diff
@@ -50,6 +52,9 @@ namespace Vls.Formatter {
             if (stderr_buf != null && stderr_buf.strip ().length > 0) {
                 throw new FormattingError.FORMAT ("%s", stderr_buf);
             } else {
+                foreach (var arg in args)
```

Comment:
I think we should find a better way to log these messages. Also, don't use `stderr.printf()`